user/downloading-installing-upgrading/installation-guide.md

```diff
@@ -19,111 +19,295 @@ redirect_from:
-ISO. Make sure to read our guide on [verifying signatures] for more info about
-how to download and verify our PGP keys and verify the downloaded ISO.
+On Windows, you can use the [Rufus] tool to write the ISO to a USB key. MediaTest is not recommended. Be sure to select "DD image" mode (you need to do that **after** selecting the Qubes ISO):
 
+<img src="/attachment/wiki/InstallationGuide/rufus-main-boxed.png" height="350">
 
-Copying the ISO onto the installation medium
---------------------------------------------
+If you are an advanced user and you would like to customize your installation, please see [Custom Installation]. Otherwise, follow the instructions below.
 
-Once you verify this is an authentic ISO, you should copy it onto a sufficiently large installation medium, such as a USB drive, dual-layer DVD, or Blu-ray disc.
-(The size of each Qubes ISO is listed on the [downloads] page.)
-Note that there are [security considerations] to keep in mind when choosing an installation medium.
+<div class="alert alert-info" role="alert">
+    <i class="fa fa-question-circle"></i>
+    <b>Note : </b> This guide will demonstrate a simple installation using mostly default settings. 
+</div>
 
-If you're using a USB drive, then you just need to copy the ISO onto the
-USB device, e.g. using `dd`:
+Installation
+------------
 
-    dd if=Qubes-R3-x86_64.iso of=/dev/sdX bs=1048576 && sync
+<h3>Getting to the boot screen</h3>
 
-Change `Qubes-R3-x86_64.iso` to the filename of the version you're installing,
-and change `/dev/sdX` to the correct target device (e.g., `/dev/sdc`).
-**Warning:** Choosing the wrong device could result in data loss. Make sure to
-write to the entire device (e.g., `/dev/sdc`) rather than just a single
-partition (e.g., `/dev/sdc1`).
+Just after you power on your machine, make the Qubes OS medium available to the computer by inserting the DVD or USB key you have previously copied the 
+Qubes OS image to. Shortly after the Power-on self-test (POST) is completed, you should be greeted with the Qubes OS boot screen. 
 
-On Windows, you can use the [Rufus] tool. Be sure to select "DD image" mode (you
-need to do that **after** selecting the Qubes ISO):
+<img src="/attachment/wiki/InstallationGuide/boot-screen.png">
 
-**Warning:** If you do that on Windows 10, you can only install Qubes without 
-MediaTest, which isn't recommended. 
+From there, you can navigate the boot screen using the arrow keys on your keyboard. Pressing the "Tab" key will reveal options. You can choose one of three options : install Qubes OS ; test this media and install Qubes OS ; troubleshooting. Select the option to test this media and install Qubes OS. 
 
-<img src="/attachment/wiki/InstallationGuide/rufus-main-boxed.png" height="350">
+If the boot screen does not appear, there are several options to troubleshoot. First, try rebooting your computer. If it still loads your currently installed operating
```

Comment:
On UEFI install there is no boot menu on R4.0 by design. It goes straight to the installer (welcome and language selection). The boot menu will be back in R4.1.

Reply:
I have added a note